src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -982,9 +982,11 @@ IdleConnectionResiliency getSessionRecovery() {
 
     /** global system ColumnEncryptionKeyStoreProviders */
     static Map<String, SQLServerColumnEncryptionKeyStoreProvider> globalSystemColumnEncryptionKeyStoreProviders = new HashMap<>();
+    static boolean isWindows;
 
     static {
-        if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).startsWith("windows")) {
+        isWindows = System.getProperty("os.name").toLowerCase(Locale.ENGLISH).startsWith("windows");
```

Comment:
Is `isWindows` used anywhere else? No need to create a new variable if its only used in one spot.